addons/l10n_syscohada/data/account.account.template.csv

```diff
@@ -1130,4 +1130,4 @@
 "pcg_96","COMPTES D'ECARTS SUR COUTS PREETABLIS","96","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
 "pcg_97","COMPTES DE DIFFERENCES DE TRAITEMENT COMPTABLE","97","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
 "pcg_98","COMPTES DE RESULTATS","98","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
-"pcg_99","COMPTES DE LIAISONS INTERNES","99","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
\ No newline at end of file
+"pcg_99","COMPTES DE LIAISONS INTERNES","99","l10n_syscohada.account_type_special","l10n_syscohada.syscohada_chart_template","False"
```

Comment:
unnecessary diff?

Reply:
There was a missing empty line at the end of the file, i think git recognize that as a diff